sdk/core/azure-core/src/main/java/com/azure/core/implementation/logging/LoggingUtil.java

```diff
@@ -0,0 +1,30 @@
+package com.azure.core.implementation.logging;
+
+import java.util.Arrays;
+
+public class LoggingUtil {
```

Comment:
This [PR](https://github.com/Azure/azure-sdk-for-java/pull/25433/files#diff-9931e319723fb2b00807723723fffafc37f496bde486fe9d231ed8e8c88226a7R11) added `LoggingUtils`. We can move these methods to the same class. I think because this is singular and the other one is plural, the PR won't complain about merge conflict.